src/_posts/platform/deployment/buildpacks/2000-01-01-apt.md

```diff
@@ -66,8 +66,9 @@ http://downloads.sourceforge.net/project/wkhtmltopdf/0.12.1/wkhtmltox-0.12.1_lin
 ```
 
 {% warning %}
-Current base image is based on **Ubuntu 18:04 Bionic**, you have to make sure
-the chosen packages are compatible with this release. See [this blog article](https://scalingo.com/articles/2019/08/06/multiple-stacks.html) to see how to change your stack.
+You have to make sure the chosen packages are compatible with the release the stack image.
+See [our documentation page]({% post_url platform/internals/stacks/2000-01-01-stacks %})
+to see how to change your stack and the **version** of the current base image.
```

Comment:
I don't understand the part of this sentence "and the **version** of the current base image."